examples/website/src/shared/route/router.tsx

```diff
@@ -91,6 +102,20 @@ function contentForRoute(activeRoute: Route | null) {
     case RouteType.Tag:
       return <TagTemplateContainer tag={activeRoute.params.tag} />
 
+    case RouteType.Login:
+      return <Login />
+
+    case RouteType.Logout:
+      const [logout] = useMutation(LogoutMutation)
+      const {session} = useContext(AuthContext)
+      const authDispatch = useContext(AuthDispatchContext)
+      if (session) {
+        logout()
+        localStorage.removeItem(LocalStorageKey.SessionToken)
+        authDispatch({type: AuthDispatchActionType.Logout})
+      }
+      return <PageTemplateContainer slug={''} />
```

Comment:
Just to understand, is this one will redirect to index page?

Reply:
Yes.